### PR TITLE
feat(rum): import sampleRUM from helix-rum-js package

### PR DESCRIPTION
**Pull request by chicharr**
## Description
Helix-rum code resides in its own repository https://github.com/adobe/helix-rum-js. So far the code is duplicated in `aem-lib` repository. 
With the changes in this PR, we pull the `helix-rum-js` code from npm, instead of copying and pasting it.
By using the `nodeResolve` rollup plugin we embed, the pulled code is embedded into `aem.js`.

## Related Issue

#2 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "@babel/core": "7.22.17",
     "@babel/eslint-parser": "7.22.15",
     "@esm-bundle/chai": "4.3.4-fix.0",
+    "@rollup/plugin-node-resolve": "^15.2.1",
```
Comment (kptdobe) [inline package.json:30]:
```suggestion
    "@rollup/plugin-node-resolve": "15.2.1",
```

Comment (kptdobe) [reply package.json:30]: Please use strict dependencies.

File: package.json [hunk 2]
```diff
@@ -42,5 +43,8 @@
     "sinon": "15.0.1",
     "stylelint": "15.10.3",
     "stylelint-config-standard": "30.0.1"
+  },
+  "dependencies": {
+    "@adobe/helix-rum-js": "^1.4.0"
```
Comment (kptdobe) [inline package.json:48]:
```suggestion
    "@adobe/helix-rum-js": "1.4.0"
```

**Review by kptdobe — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (chicharr) [reply package.json]: DONE

File: package.json (hunk 1 above)
Comment (trieloff) [reply package.json]: Renovatebot does this by default with the first PR it creates.

**Review by trieloff — DISMISSED** [review]

**Comment by chicharr** [comment]
Switched to strict dependencies, and updated to latest `rum.js` version available

**Review by trieloff — APPROVED** [review]

**Review by kptdobe — APPROVED** [review]

**Comment by kptdobe** [comment]
Ready to merge ?

**Comment by chicharr** [comment]
> Ready to merge ?

Yes, shall I?

**Pull request merged**